Configuration/TCA/tx_news_domain_model_news.php

```diff
@@ -492,7 +492,8 @@
                 'allowed' => 'tx_news_domain_model_tag',
                 'MM' => 'tx_news_domain_model_news_tag_mm',
                 'foreign_table' => 'tx_news_domain_model_tag',
-                'foreign_table_where' => 'ORDER BY tx_news_domain_model_tag.title',
+                'foreign_table_where' => ' AND (tx_news_domain_model_tag.sys_language_uid = 0 OR tx_news_domain_model_tag.l10n_parent = 0) ORDER BY tx_news_domain_model_tag.title',
```

Comment:
Would not it have to be this? tx_news_domain_model_tab.sys_language_uid IN (-1,0)

Reply:
I would go with the uid in(). wanna change that?

Reply:
> Would not it have to be this? tx_news_domain_model_tag.sys_language_uid IN (-1,0) or better ###REC_FIELD_sys_language_uid### as placeholder?

I took the configuration from the categories field, but if the in()-approach is better, I'll fix it.

Is this correct?
`'foreign_table_where' => ' AND (tx_news_domain_model_tag.sys_language_uid IN (-1,0) OR tx_news_domain_model_tag.l10n_parent = 0) ORDER BY tx_news_domain_model_tag.title',`